sig-docs/english-doc-list/KubeSphere 3.0 Documentation List (DevOps and Built-in Apps).md

```diff
@@ -53,8 +53,8 @@ As for the apps documentation, in addition to basic deployment on KubeSphere, So
 | Deploy Redis Exporter in KubeSphere         |        |         |       |  
 | Deploy MySQL Exporter in KubeSphere         |  Guangzhe Huang      |    https://kubesphere.io/docs/project-user-guide/custom-application-monitoring/get-started/monitor-mysql/     |
 | Deploy RabbitMQ in KubeSphere               | wenxin       |         |       |  
-| Deploy Memcached in KubeSphere              |        |         |       |  
-| Deploy PostgreSQL in KubeSphere             |        |         |       |  
+| Deploy Memcached in KubeSphere              |  Yunkun Rao      |         |       |
```

Comment:
Please fill out the language you choose to write the document, thanks.

Reply:
got it